registry/api.py

```diff
@@ -0,0 +1,99 @@
+from http.client import OK
+import json
+from msilib.schema import Registry
+from typing import Optional
+from urllib.request import Request
+from fastapi import APIRouter, Depends
+from requests import request
+import requests
+from access_control.access import global_admin_access, get_user, project_read_access, project_write_access
+from access_control.models import User
+from access_control.db_rbac import DbRBAC
+
+router = APIRouter()
+rbac = DbRBAC()
+registry_url = "https://feathr-sql-registry.azurewebsites.net/api/v1"
```

Comment:
Please make this a server side configuration

Reply:
move to config.REGISTRY_URL